<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchItemDialogClass</class>
 <widget class="QDialog" name="SearchItemDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>376</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SearchItemDialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>20</y>
     <width>351</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="noButton">
      <property name="text">
       <string>无筛选</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="titleButton">
      <property name="text">
       <string>按标题筛选</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="titleLine"/>
    </item>
    <item>
     <widget class="QRadioButton" name="idButton">
      <property name="text">
       <string>按 ID 筛选</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="idLine"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>330</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
